packages/analytics-js-integrations/src/integrations/VWO/browser.js

```diff
@@ -42,6 +42,17 @@ class VWO {
     } else {
       logger.debug('===[VWO]loadIntegration flag is disabled===');
     }
+
+    window.VWO = window.VWO || [];
+    window.VWO.event = window.VWO.event || function () {
+        window.VWO.push(['event'].concat([].slice.call(arguments)));
+    };
+ 
+    
+    window.VWO.visitor = window.VWO.visitor || function () {
+        window.VWO.push(['visitor'].concat([].slice.call(arguments)));
```

Comment:
Same as above, Use rest params instead of 'arguments'

Reply:
Resolved